tools/install.sh

```diff
@@ -86,10 +86,15 @@ mv -f ~/.zshrc-omztemp ~/.zshrc
 # If this user's login shell is not already "zsh", attempt to switch.
 TEST_CURRENT_SHELL=$(expr "$SHELL" : '.*/\(.*\)')
 if [ "$TEST_CURRENT_SHELL" != "zsh" ]; then
-  # If this platform provides a "chsh" command (not Cygwin), do it, man!
-  if hash chsh >/dev/null 2>&1; then
+  # If this platform provides a "chsh" command (not Cygwin) and ZSH is installed
+  # globally, do it, man!
+  if hash chsh >/dev/null 2>&1 && [ "$CHECK_ZSH_INSTALLED" -ge 1 ]; then
     printf "${BLUE}Time to change your default shell to zsh!${NORMAL}\n"
     chsh -s $(grep /zsh$ /etc/shells | tail -1)
+  # If ZSH only installed non-globally.
+  elif [ "$CHECK_ZSH_INSTALLED_NON_GLOBALLY" -eq 0 ] && [ ! "$CHECK_ZSH_INSTALLED" -ge 1 ]; then
+    printf "I can't change your shell automatically because ZSH is not installed globally.\n"
```

Comment:
this should maybe say "...or your system does not support `chsh`." (like cygwin, which doesn't have chsh, because... well, reasons.) May be easier to just leave out the "...because..." part? On Cygwin and similar systems, this would give a misleading diagnostic message.